Added blitz code for a faster playthrough.

phku-hk opened the pull request.

Commit:
Added blitz code for a faster playthrough.

phku-hk committed:
Updated README

Commit:
Revert "Updated README"

This reverts commit e9928639c8bb4c8c1964da6ca26b6db19e7387d6.

abyssonym commented on tables/requirements_blitz.txt:
```diff
@@ -0,0 +1,95 @@
+lisa				arrowlike
+marie				breakable
+clare				party1
+daos_shrine         lisa&marie&clare&any_key
```
It is not possible to fly to Doom Island without the engine; you should either add the engine back as a requirement to reach Daos Shrine, or create a modified event that does not require the engine.

phku-hk replied:
I assumed that was just a cutscene. sorry, my bad. I will try to create a modified event.

abyssonym commented on tables/requirements_blitz.txt:
```diff
@@ -0,0 +1,95 @@
+kamirno             airship&party3
+lisa				arrowlike
+marie				breakable
+clare				party1
```
I'm not sure what is achieved by creating "lisa", "marie", and "clare" as locations. I don't know what your intent is for these lines, but I suspect that a required item might end up at one of these locations, which will make the seed unbeatable. For example, what if the bombs end up at the "lisa" location, which are then required to unlock the rest of the seed?

phku-hk replied:
I assumed that file defines routing requirements, so I added those maides. Also somewhere in your code* it tests the seed, so I did not assume any problems there. Also several people tested these requirements and had no problems. I will try to find such a "dead lock" you mentioned.

*itemrouter.py:703: success = self.try_unlock_locations(self.goal_requirements)

Commit:
Fixed not being able to fly to doom island without engine.
Modified routing requirements.

phku-hk commented:
I fixed not being able to fly to doom island and tested it using a quick seed. This only affects blitz mode. You still need engine in non-blitz modes. 
Also I changed blitz mode routing requirements to make it more interesting.

Commit:
Added.

Commit:
Updated

phku-hk committed:
Create python-packaging.yml

phku-hk committed:
Update python-packaging.yml

phku-hk committed:
Create python-build-exe.yml

Commit:
Adjusted Blitz Mode Seed Generation
Updated Ideas
Updated Blitz Routing Requirements

phku-hk committed:
Delete .github/workflows directory

Commit:
Some corrections

Commit:
Blitz seed now always have between 20 to 25 checks for a fast playthrough.
Also changed requirements to add at least one capsule monster.

Commit:
Changed version string to show blitz.

Commit:
Changed version string.

Commit:
Added secret code: 'sinistrals'
Spoiler now shows bosses.

Commit:
Added 'last_sinistral' secret code

Commit:
Added secret code 'music_last_sinistral' to make battle theme 2 play at the final boss.
Fixed sinistrals not showing up in battle.
